types/openapi-v2/index.d.ts

```diff
@@ -0,0 +1,1134 @@
+     * When defined alongside type being array (outside the items), it will affect the wrapping element and only if wrapped is true. If wrapped is false, it will be ignored.
+     */
+    name?: string;
+    /**
+     * The URL of the namespace definition. Value SHOULD be in the form of a URL.
+     */
+    namespace?: string;
+    /**
+     * The prefix to be used for the name.
+     */
+    prefix?: string;
+    /**
+     * Declares whether the property definition translates to an attribute instead of an element. Default value is false.
+     */
+    attribute?: boolean;
+    /**
+     * MAY be used only for an array definition.
+     * Signifies whether the array is wrapped (for example, <books><book/><book/></books>) or unwrapped (<book/><book/>).
+     * Default value is false. The definition takes effect only when defined alongside type being array (outside the items).
+     */
+    wrapped?: boolean;
+}
+
+/**
+ * The Schema Object allows the definition of input and output data types.
+ * These types can be objects, but also primitives and arrays.
+ * This object is based on the JSON Schema Specification Draft 4 and uses a predefined subset of it.
+ * On top of this subset, there are extensions provided by this specification to allow for more complete documentation.
+ *
+ * Further information about the properties can be found in JSON Schema Core and JSON Schema Validation.
+ * Unless stated otherwise, the property definitions follow the JSON Schema specification as referenced here.
+ *
+ * https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#schema-object
+ */
+export interface OpenApiSchema {
+    /**
+     * External definition
+     */
+    $ref?: string;
+    /**
+     * The title of the schema.
+     */
+    title?: string;
+    /**
+     * A short description of the schema.
+     */
+    description?: string;
+    /**
+     * Declares the value of the header that the server will use if none is provided.
+     * (Note: "default" has no meaning for required headers.) See https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-6.2.
+     * Unlike JSON Schema this value MUST conform to the defined type for the header.
+     */
+    default?: any;
+    /**
+     * See https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-5.1.2.
+     */
+    maximum?: number;
+    /**
+     * See https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-5.1.2.
+     */
+    exclusiveMaximum?: boolean;
+    /**
+     * See https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-5.1.3.
+     */
+    minimum?: number;
+    /**
+     * See https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-5.1.3.
+     */
+    exclusiveMinimum?: boolean;
+    /**
+     * 	See https://tools.ietf.org/html/draft-fge-json-schema-validation-00#section-5.2.1.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/openapi-v2/index.d.ts;
  Line: 139;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.